src/Socket.hpp

```diff
@@ -42,6 +42,7 @@ class Socket {
   double timeout_;
   bool data_written_;
   std::string response_;
+  // std::string request_;
```

Comment:
i'm not sure why this is commented out instead of deleted - can we kick it?

Reply:
Hole main loop is still to be updated. And request_ is a reminder to be added.